airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -1113,6 +1113,9 @@ class WorkflowRuns(SemiIncrementalMixin, GithubStream):
     # key for accessing slice value from record
     record_slice_key = ["repository", "full_name"]
 
+    # https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs
+    re_run_period = 32 # days
```

Comment:
why 32 and not 30?

Reply:
I have played with this 30 days border edge and I found a couple of things:

1. Objects are "almost" sorted - from time to time some near objects are out of order. I think it's internal details of the GitHub messenging queue.
But it's no big deal because it's related to pretty close objects.

2. I have also played with object mutation on this 30 days border and it looks like this border is blurred a little bit.

I just decided to add a couple of days to get a 100% guarantee that the object will not mutate anymore.